v2/cmd/asoctl/cmd/export_template.go

```diff
@@ -6,14 +6,22 @@
 package cmd
 
 import (
+	"fmt"
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"os"
+	"reflect"
 	"regexp"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	. "github.com/Azure/azure-service-operator/v2/internal/logging"
```

Comment:
minor: I think this line can/should be in the 3rd stanza below (lines 22-24) as it's from aso

Reply:
It's a `.` import, so I separated it for better visibility so readers know it's special.